api/core/tools/provider/builtin/tavily/tools/tavily_search.yaml

```diff
@@ -25,3 +25,138 @@ parameters:
       pt_BR: used for searching
     llm_description: key words for searching
     form: llm
+  - name: search_depth
+    type: string
```

Comment:
Oops, there is still a type need to be change to `select`

Reply:
sorry for having soooo many errors and make this PR been this long. i'm kinda busy with school stuff these days so you know😭😭😭😭
but i have change the type already!